config-db.yml

```
@@ -45,6 +45,15 @@
         - lookup('env', 'PRIMARY_IP_ADDR') is not defined or lookup('env', 'PRIMARY_IP_ADDR') | length == 0
       tags: primary-db
 
+    - include_role:
+        name: workload-agent
+      when:
+        - ansible_facts['product_name'] == "Google Compute Engine"
```

If not running on Google Cloud, this step will be silently skipped;  maybe worth a warning or message of some type?

---

Done